client/app/components/parameter-settings.html

```diff
@@ -13,6 +13,7 @@ <h4 class="modal-title">{{$ctrl.parameter.name}}</h4>
         <select ng-model="$ctrl.parameter.type" class="form-control">
           <option value="text">Text</option>
           <option value="number">Number</option>
+          <option value="enum">Enum</option>
```

Comment:
I don't think so, but this text is exposed to people writing queries only, will non-devs be writing queries often? I would think if you can write a query that you would know what an enum is. MySQL at least has a [Enum type](https://dev.mysql.com/doc/refman/5.7/en/enum.html). I'm fine to rename it if you'd like